script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1391,109 +1586,133 @@ I don't give a damn whether she was being serious or just trying to rile me up.
 [sha:d01136379096ad8f457455cf8bf12aac76236456]{
 -- Page 252, Offset 22786.
 -- 先輩にとってアルクェイドは倒すだけの吸血鬼で、そんなものを信じる俺は愚か者だと思っている。
-To Senpai, Arcueid is nothing more than a vampire that must be defeated, and she thinks I'm a fool for trusting her.
+It's clear that either way,
+ Senpai thinks Arcueid is nothing more than a vampire to be defeated,
+ and that I'm a fool for trusting her.
```

Comment:
There's a slight pronoun confusion here that might be possible to avoid, because it could be read as Shiki is a fool for trusting Ciel.
It's pretty minor, but generally good practice.

>...Senpai thinks I'm a fool for trusting Arcueid, who she sees as nothing more than a vampire to be defeated.